adapter/pkg/synchronizer/types.go

```diff
@@ -64,6 +64,7 @@ type GatewayLabel struct {
 	Name              string `json:"name"`
 	Vhost             string `json:"vhost"`
 	DeployedTimeStamp int64  `json:"deployedTimeStamp"`
+	DeploymentType    string `json:"deploymentType"`
```

Comment:
Can't we use this as a boolean type ? Since we are considering only two values.

Reply:
This value is populated from the deployment.json file straightaway
Hence it does not need a boolean type

Reply:
Although we have two values right now (production, sandbox), its possible that "development" will be introduced in the future